Formula/countdown.rb

```diff
@@ -0,0 +1,17 @@
+class Countdown < Formula
+  desc "Terminla countdown timer"
+  homepage "https://github.com/antonmedv/countdown"
+  url "https://github.com/antonmedv/countdown/archive/refs/tags/v1.0.0.tar.gz"
+  sha256 "695bb6a57d7ba5a700ac16a7676132ee61c786a5e9a1ca774607d6f2ad5f39d7"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w")
+  end
+
+  test do
+    pipe_output bin/"countdown", "0m0s"
```

Comment:
```suggestion
    system bin/"countdown", "0m0s"
```
You're not doing anything with the output (or even piping anything to `countdown`), so we don't need `pipe_output` here.

Reply:
yeah, but we ran into some test failure as below

```
==> /opt/homebrew/Cellar/countdown/1.0.0/bin/countdown 0m0s panic: termbox: error while reading terminfo data: EOF goroutine 1 [running]: main.main() github.com/antonmedv/countdown/main.go:78 +0x2e4 Error: countdown: failed An exception occurred within a child process: BuildError: Failed executing: /opt/homebrew/Cellar/countdown/1.0.0/bin/countdown 0m0s
```